src/discof/gossip/fd_gossip_tile.c

```diff
@@ -28,27 +28,35 @@ typedef struct {
 } fd_gossip_out_ctx_t;
 
 struct fd_gossip_tile_ctx {
-  fd_gossip_t * gossip;
+  fd_gossip_t *        gossip;
 
-  fd_pubkey_t   identity_key[1]; /* Just the public key */
+  fd_contact_info_t    my_contact_info[1];
 
-  uint          rng_seed;
-  ulong         rng_idx;
+  uint                 rng_seed;
+  ulong                rng_idx;
 
-  double        ticks_per_ns;
-  long          last_wallclock;
-  long          last_tickcount;
+  double               ticks_per_ns;
+  long                 last_wallclock;
+  long                 last_tickcount;
 
-  uchar         buffer[ FD_NET_MTU ];
+  uchar                buffer[ FD_NET_MTU ];
 
-  fd_gossip_in_ctx_t in[ 32UL ];
-  int                in_kind[ 32UL ];
+  fd_gossip_in_ctx_t   in[ 32UL ];
+  int                  in_kind[ 32UL ];
 
-  fd_gossip_out_ctx_t net_out[ 1 ];
-  fd_gossip_out_ctx_t gossip_out[ 1 ];
+  fd_gossip_out_ctx_t  net_out[ 1 ];
+  fd_frag_meta_t *     net_out_mcache;
+  ulong                net_out_seq;
+  ulong                net_out_depth;
+
+  fd_gossip_out_ctx_t  gossip_out[ 1 ];
+  fd_gossip_out_ctx_t  sign_out[ 1 ];
 
   fd_keyguard_client_t keyguard_client[ 1 ];
   fd_keyswitch_t *     keyswitch;
+
+  fd_ip4_udp_hdrs_t    net_out_hdr[ 1 ]; /* Used to construct outgoing network packets */
+  ushort               net_id;           /* Network ID for outgoing packets */
```

Comment:
Really don't like these kind of comments. "Network ID for outgoing packets" is no more useful than "net_id". If you're going to  comment these, actually write a comment,

```suggestion
  /* The IPv4 protocol which we send outgoing UDP packets with requires a ushort identification
      field in the header, which should be "unique", although this requirement is no longer relevant
      or enforced in modern networking. We simply increment it for every outgoing packet. */
  ushort               net_id;           /* Network ID for outgoing packets */
```

Reply:
(You can also not write a comment at all, which is fine, since the use of these fields is pretty obvious)

Reply:
yeah sorry about these half of them are just copilot remnants